index.html

```diff
@@ -176,7 +176,7 @@
   <p>
     The term <dfn>obtain permission</dfn> for a certain operation indicates
     that the UA has either obtained <a>expressed permission</a>, or
-    <a href="#askforgiveness">asked for forgiveness</a>, or ensured a
+    <a href="#dfn-ask-for-forgiveness">asked for forgiveness</a>, or ensured a
```

Comment:
```suggestion
    <a>asked for forgiveness</a>, or ensured a
```

Reply:
The `dfn` is "ask for forgiveness",  use `<a>asked for forgiveness</a>` will throw ReSpec error.

Reply:
ah, change the dfn to:
```
<dfn data-lt="asked for forgiveness">ask for forgiveness</dfn> 
```

Then you can use the alias "asked for forgiveness" :)